chaoscenter/cypress/e2e/UserManagment.cy.js

```diff
@@ -0,0 +1,105 @@
+describe('testing UserManagement', () => {
+    before(() => {
+        cy.requestLogin(Cypress.env('username'),Cypress.env('password'));
+    });
+
+    it('tesing through REST APIs', () => {
+        const accessToken = localStorage.getItem('accessToken');
+
+        //add user
+        const add_payload = {
+            name: 'user10',
+            email: 'user10@gmail.com',
+            username: 'user10',
+            password: 'user10',
+            role: 'user'
+          };
+
+        cy.request({
+        method: 'POST',
+        url: '/auth/create_user', 
+        headers: {
+            Authorization: `Bearer ${accessToken}`
+        },
+        body: add_payload,
+        }).then((response) => {
+            expect(response.status).to.equal(200);
+            expect(response.body.username).to.eq(payload.username);
+            expect(response.body.email).to.eq(payload.email);
+            expect(response.body.name).to.eq(payload.name);
+        });
+
+        //edit password
```

Comment:
@Dhanush0369 Can we shift this into it's own it() block.

Structure we can follow could be either - 

```
describe("API Tests"){
  it(create)
  it(edit)
  it(delete)
}

describe("UI Tests"){
  it(create)
  it(edit)
  it(delete)
}
```

OR

Have UI tests in one file & API tests in other.